src/openrct2-win/openrct2-win.cpp

```diff
@@ -48,33 +48,26 @@ int wmain(int argc, wchar_t** argvW, [[maybe_unused]] wchar_t* envp)
 static char** GetCommandLineArgs(int argc, wchar_t** argvW)
 {
     // Allocate UTF-8 strings
-    auto argv = (char**)malloc(argc * sizeof(char*));
-    if (argv != nullptr)
```

Comment:
Any reason for dropping this check?

Reply:
It's no longer valid. Unless you use noexcept new, it will throw with std::bad_alloc instead of returning nullptr.